scripts/globals/items/fishermans_belt.lua

```diff
@@ -0,0 +1,35 @@
+-----------------------------------
+-- ID: 15452
+-- Item: Fisherman's belt
+-- Enchantment: Fishing image support
+-- 2 hours, All Races
+-----------------------------------
+-- Enchantment: Fishing image support
+-- Duration: 2 hours
+-- Fishing Skill +2
+-----------------------------------
+require("scripts/globals/status")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    local result = 0
+    if target:hasStatusEffect(xi.effect.FISHING_IMAGERY) == true then
+        result = xi.effect.FISHING_IMAGERY
```

Comment:
Yeah, this opened up a can of worms. All of the crafting belts are set to the wrong message. Began looking for the DAT with the messages in them, but no luck so far. Otherwise, going through hundreds of numbers, testing each one manually..

Reply:
whats the text you see in game

Reply:
![image](https://user-images.githubusercontent.com/61239975/120869215-f18e4880-c563-11eb-99d7-b49db1850f21.png)
![image](https://user-images.githubusercontent.com/61239975/120869221-f5ba6600-c563-11eb-8cd2-3aeec3eda0d8.png)

Reply:
The existing belts and support system need... work as well. Yay, worms.

Reply:
oh thats gonna be fun: its gonna need a param sent for the message to know which one its waiting on expiring. I don't think the core end of this expects that. May have to return a non message and trip a messageBasic() to do it.

Reply:
@Era-Lusiphur Is this still alive? :)

Reply:
Thought I posted a reply here yesterday... Waiting to make a global to handle existing imagery effects issues.